<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCommandLinkButton" name="openSqlDatabaseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть базу данных SQL</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="openCvsDatabaseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть базу данных CVS</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="openTvsDatabaseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть базу данных TVS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>29</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>DATABASE MANAGER</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Открыть уже существующую базу данных...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Создать новую базу данных...</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="createCvsDatabaseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать базу данных CVS</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="createTvsDatabaseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать базу данных TVS</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="createSqlDatabaseButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать базу данных SQL</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Продолжить без шагов</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
